docs/publish/centralized-deployment.md

```diff
@@ -73,7 +73,7 @@ If your organization meets all requirements, complete the following steps to pub
     ![Edit who has access page in Office 365 admin center](../images/power-bi-tiles-edit.png)
 
     > [!NOTE]
-    > For single sign-on (SSO) add-ins, the users and groups assigned will also be shared with add-ins that share the same Azure App ID. Any changes to user assignments will also apply to those add-ins. The related add-ins will be shown on this page. For SSO add-ins only, this page will display the list of Microsoft Graph permissions that the add-in requires.
+    > A [single sign-on (SSO)](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins) system for add-ins is currently in preview and should not be used for production add-ins. When an add-in using SSO is deployed, the users and groups assigned will also be shared with add-ins that share the same Azure App ID. Any changes to user assignments will also apply to those add-ins. The related add-ins will be shown on this page. For SSO add-ins only, this page will display the list of Microsoft Graph permissions that the add-in requires.
```

Comment:
I'm wondering if we should use the future tense here. Does centralized deployment support this scenario today? If so, I think this should be present tense.